### DISCUSSION TITLE
How to link station location logbooks

### DISCUSSION THREAD

**Discussion by dpezet** [wavelog/wavelog#597] [Q&A] [answered]
I have created multiple station locations for my primary logbook (one for each POTA park I have activated). However, I cannot see how to link them together for reporting/analysis. I see the "Linked" column in the list, but I don't see any option to actually link them together. Could someone help point me in the right direction? Thank you!
![image](https://github.com/user-attachments/assets/88cedb38-4b6b-4c9a-8e94-ac038219757c)

**Comment by dpezet** [comment, marked as answer]
Never mind. I found it under the station setup. Here it is for anyone else looking for it. 
![image](https://github.com/user-attachments/assets/feff4f2a-3599-448d-b375-781304a4bef1)

**Reply by AndreasK79** [reply]
Happy you found the solution. We try to make everything easy to use, so feel free to make suggestions.

**Reply by om1cmd** [reply]
I just went through the same process and wanted to link the new location to my logbook but had to google the solution and found this thread. I suggest adding one more line to the station location description that mentions how to do this. Linking is mentioned for the first time in this description and I found it confusing that I could not see any linking settings in station location setup.
<img width="719" height="154" alt="Screenshot from 2025-08-31 18-10-14" src="https://github.com/user-attachments/assets/a9741b2b-811b-4014-b87e-b11b70ff2d23" />
I suggest adding something like: "To link Station Locations to a Station Logbook use Linked Locations in Station Logbook table."

**Reply by AndreasK79** [reply]
@om1cmd I see the problem, but I got a simpler solution:
<img width="224" height="288" alt="image" src="https://github.com/user-attachments/assets/165ca16b-0681-4525-9c29-5c4abaa46966" />

By adding the "Edit" word, it should be more obvious.
